packages/manager/apps/dedicated/client/app/billing/payment/credits/translations/Messages_fr_FR.json

```diff
@@ -14,12 +14,25 @@
   "billing_credit_balance_movements_details": "Détails des mouvements",
   "billing_credit_balance_movements_title": "Détails des mouvements de votre bon d'achat {{t0}}",
   "billing_credit_balance_movements_back": "Retour à mes bons d'achat",
+  "billing_credit_balance_movements_header_id": "ID",
   "billing_credit_balance_movements_header_date": "Date",
   "billing_credit_balance_movements_header_operation": "Opération",
   "billing_credit_balance_header_order": "Commande",
   "billing_credit_balance_movements_operation_credit": "Crédit",
   "billing_credit_balance_movements_operation_debit": "Débit",
   "billing_credit_balance_movements_operation_debit_order": "Débit sur la commande <a href=\"{{orderUrl}}\" target=\"_blank\">{{ orderId }}</a>",
+  "billing_credit_balance_movements_operation_debit_refund": "Émission de l'avoir <a href=\"{{refundUrl}}\" target=\"_blank\">{{ refundId }}</a>",
   "billing_credit_balance_movements_load_error": "Une erreur est survenue lors du chargement des mouvements de votre bon d'achat.",
-  "billing_credit_back": "Retourner à mes bons d'achat"
+  "billing_credit_back": "Retourner à mes bons d'achat",
+  "billing_credit_sbg_title": "Opérations concernant vos mesures commerciales SBG",
+  "billing_credit_sbg_total_amount": "Montant global des mesures commerciales : {{amount}}",
+  "billing_credit_sbg_total_balance": "Solde du bon d'achat au {{date}} : {{amount}}",
+  "billing_credit_sbg_services_intro": "Vous bénéficiez de mesures commerciales* relatives à votre (vos) service(s) impactés par l’incendie survenu sur notre site de Strasbourg (SBG). ",
```

Comment:
```suggestion
  "billing_credit_sbg_services_intro": "Vous bénéficiez de mesures commerciales* relatives à votre (vos) service(s) impacté(s) par l’incendie survenu sur notre site de Strasbourg (SBG). ",
```

Reply:
Thanks for the update. It will be updated on a second step.